src/pykeen/nn/node_piece.py

```diff
@@ -841,10 +844,13 @@ def resolve_aggregation(
     :return:
         the chosen aggregation function.
     """
+    # Create an MLP for string aggregation
     if aggregation is None:
         return torch.mean
 
     if isinstance(aggregation, str):
+        if aggregation == "mlp":
```

Comment:
this will cause clashes with the other PR where I replaced this function with a FunctionResolver

Reply:
but I'm not against doing this in principle. we could also introduce the deepset encoder from @migalkin's example and use this function to handle adding that

Reply:
I am also okay with bringing the other PR into `master` first

Reply:
I just need to type edit Mikhail's text first... I'm trying to finish the paper edits so ben will be happy when we have our team meeting in an hour